pypit/arms.py

```diff
@@ -62,6 +64,9 @@ def ARMS(spectrograph, fitstbl, setup_dict):
     # Init calib dict
     calib_dict = {}
 
+    # Spectrometer class
+    # This will be passed into the Calibrations class in PR #368
```

Comment:
I think this comment about the PR can be removed

Reply:
removed